.github/workflows/pr.yaml

```diff
@@ -35,6 +36,42 @@ jobs:
       - devcontainer
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.10
+    if: always()
+    with:
+      needs: ${{ toJSON(needs) }}
+  changed-files:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/changed-files.yaml@branch-24.10
+    with:
+      files_yaml: |
```

Comment:
I think `!.devcontainers/**` could also safely be added to these lists. A change that just updates the base image for devcontainers shouldn't require running any of the other tests in this repo.

Reply:
I've added `!.devcontainers/**`.